### PR TITLE
add funding links to composer.json and github config

### PR DESCRIPTION
**Pull request by Flyingmana**
following the discussion in https://github.com/OpenMage/magento-lts/discussions/3495#discussioncomment-7541893 we created an openCollective account to make donations to the project possible.

A rough planning document for making use of the founding can be found here https://github.com/OpenMage/organizational/blob/main/finance/expense_planning.md

There might be followup PRs to put the reference to openCollective also in other parts.

### CODE REVIEW

**Review by sreichel — CHANGES_REQUESTED** [review]
This needs discussed before.

**Review by sreichel — CHANGES_REQUESTED** [review]
Please wait with it.

File: .github/FOUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+open_collective: openmage
```
Comment (kiatng) [inline .github/FOUNDING.yml:2]: Should the file name be `FUNDING.yml`?

**Comment by sreichel** [comment]
With this i`d possibly loose my free phpstorm licencse. jetbrains definitly checks for it and already asked about "partners" mentioned on website!

File: .github/FOUNDING.yml (hunk 1 above)
Comment (Flyingmana) [reply .github/FOUNDING.yml:2]: it was this name when I added it for the composer installer

**Comment by Flyingmana** [comment]
> This needs discussed before.

The discussion did go 2 months public in the linked one and openCollective was suggested 2months ago.

The topic of collecting funding is additionally something we talk now internally for over 3 years (probably longer, but I did not find it in my mails yet)

In all these years openCollective was the only suggested option which was actually possible to do with our resources.

**Comment by sreichel** [comment]
Two month to make such a decision? 

Your going to sell others people work.

Who has access to that money? 
How is it distributed?

Do you pay my phpStorm license?

**Comment by Flyingmana** [comment]
for documentation why we talk about phpstorm license here

https://www.jetbrains.com/community/opensource/#support

2 of the requirements listed

> * Are not continuously funded by commercial companies or organizations (NGO, educational, research, or governmental). Compensation for server/infrastructure costs and one-time voluntary donations are allowed.
> * Are maintained by developers voluntarily in their free time. Contributors who work on the project during working hours or who engage in paid freelance work are not supported.

besides:

> You may use free licenses solely for developing non-commercial open-source projects. They cannot be used for providing paid services, your work projects, or any other commercial activities.

current pricing for Businesses is:

> per user, first year
> €249.00
> incl. VAT €296.31
> 
> second year
> €199.00
> incl. VAT €236.81
> 
> third year onwards
> €149.00
> incl. VAT €177.31

**Comment by sreichel** [comment]
@Flyingmana

I already had to deal with jetbrains support, just because auf "partners"-page at [openmage.org](url).

### Question 1

Are YOU going to pay my licence if it is not renewed after years, b/c of this?

### Question 2

Who has access to that money?

Recently I only see YOU as part of that *collective*, It seems only YOU have control about it.

Sorry for saying that ... but it looks like  **scam**! I dont know about any service OM has to pay for.

### Question 3

What happens to that money?

What happens if someone give a 10k as major sponsorship?

**Comment by kiatng** [comment]
I think opencollection is a very good step to get funding and to have some kinda future for OM. 

Question 2 of course needs to be addressed, but I have faith that the distribution of money will be transparent and fair. I also trust @Flyingmana is of good intention. There are laws governing these sort of things which I hope will not come to bear.

Question 3 really depends on the amount of funding that the project can raise. We are hoping that the fund is enough to cover maintainer's license fees etc. and may be even support a couple of fulltime positions.  

Question 1 on phpstorm sponsorship, I am not sure how many OM maintainers rely on the sponsorship that will be impacted, I use VS Code, but I would be happy to help @sreichel out on the cost of phpstorm license.

File: .github/FOUNDING.yml (hunk 1 above)
Comment (Flyingmana) [reply .github/FOUNDING.yml:2]: argh yes, now I see it too 😨

Comment (Flyingmana) [reply .github/FOUNDING.yml]: filename is corrected 🙈

**Comment by Flyingmana** [comment]
> @Flyingmana
> 
> I already had to deal with jetbrains support, just because auf "partners"-page at [openmage.org](url).
> 
> ### Question 1
> Are YOU going to pay my licence if it is not renewed after years, b/c of this?
> 

the expense planning document is linked, you can do a PullRequest there for which level this expense should get covered.


> ### Question 2
> Who has access to that money?
> 
> Recently I only see YOU as part of that _collective_, It seems only YOU have control about it.
> 

The people who get access to decide about the money are the 3 people who are listed now, more admin level maintainers might get added when they request it. But it is likely that I will be the one who primarily takes care of it.

> Sorry for saying that ... but it looks like **scam**! I dont know about any service OM has to pay for.
> 

May i say, the trick about scamming people is avoiding liability, usually by location or anonymity. Both are options I dont have. And I honestly dont see a way to avoid liability here in case of mismanagement of funds. But Iam open to hear about them to see if I can make sure they do not apply to me. 

> ### Question 3
> What happens to that money?
> 
> What happens if someone give a 10k as major sponsorship?

Again I want to point to the linked document, there is a rough plan for what the money is intended.
There is no type of Reward listed yet, as we did not define any rewards yet.

**Comment by elidrissidev** [comment]
> I already had to deal with jetbrains support, just because auf "partners"-page at [openmage.org](url).

I got asked about it twice when I applied for a license back in 2022, this year I renewed it and paid for it because I use it for commercial work.

Would a PR be accepted to remove the partners from the website that are not actually supporting the project? @Flyingmana

**Comment by Flyingmana** [comment]
> 
> Would a PR be accepted to remove the partners from the website that are not actually supporting the project? @Flyingmana

probably at one point.
But first we should figure out what kind of sponsorship we require for a listing. And also give it some time to actually have a few sponsors. But in any case, thats a topic for a different discussion.

**Comment by sreichel** [comment]
@kiatng Thank you very much ;)

**Review by fballiano — APPROVED** [review]

**Review by kiatng — APPROVED** [review]

**Review by m-overlund — APPROVED** [review]

**Pull request merged**